bin/oneoff/backfill_data/destroy_sections_and_followers

```diff
@@ -24,17 +21,17 @@ begin
       # Manually execute the `dependent: :destroy` for `has_many :sections`, including its
       # `dependent: :destroy` for `has_many :followers`. Though only teachers should have sections,
       # our DB data fails this validation, so we perform this for all users.
-      Section.where(user_id: user.id, deleted_at: nil).find_each do |section|
+      Section.where(id: user.id, deleted_at: nil).find_each do |section|
```

Comment:
Thanks for saving me. Flipped user_id to id at the wrong place. Fixed in the rebased commit.